app/reporting.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: page
+title: Reporting
+analytics_key: reporting
+parent:
+  title: How to use the pilot service
+  url: /guide
+previous:
+  title: Recording a vaccination
+  url: /recording-a-vaccination
+next:
+  title: Managing users
+  url: /managing-users
+---
+
+{% include 'guide-contents.njk' %}
+
+Vaccination data is available to extract as soon as it's recorded and saved. You can extract data in a ‘.csv format’ (comma separated variable) to support reporting within your organisation by choosing from a combination of:
+
+* Dates (up to a maximum of 2 weeks)
+* Vaccines
+* Sites
+
+You aren't restricted to how often or when you can create reports.
+
+## Choosing dates
+
+Select preset dates or a custom date range.
+
+For custom reports, you must enter a past date range, up to a maximum of 2 weeks.
```

Comment:
worth clarifying that this means a 2 week duration rather than maximum 2 weeks ago?

Reply:
The date range for custom reports is limited to 2 weeks. Dates must be in the past.